docs/sources/determine-use-case/index.md

```diff
@@ -11,32 +11,52 @@ weight: 400
+Your use case determines what’s most important.
+For example, in use case 1, if a service is misbehaving, then you might want to see the profile types so you can see the CPU and memory profiles alongside each other.
 
 For either use case, the first step is to identify areas of interest by reviewing profiles or a single service.
-Selecting different profile types lets you focus on memory allocation, CPU processes, allocation sizes, blocks, or blocks.
+Selecting different profile types lets you focus on memory allocation, CPU processes, allocation sizes, blocks, or mutually exclusive (mutex).
+
 The profile types available depend on how you've instrumented you app to generate your profiling data.
-For more information, refer to [Understand profile types](https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/profiling-types/).
+For more information, refer to [Profiling types](../concepts/#profile-types/) for help selecting a profile type to match your use case.
+Refer to [Understand profile types](https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/profiling-types/) to learn about profile types and instrumentation methods.
 
 After you’ve identified the problem process or service, you can filter and explore using labels and flame graphs to view lower levels.
+With capabilities like the Flame graph AI interpreter or the GitHub integration, Explore Profiles helps you locate the root cause and how to address it.
+
+## Use case 1: Investigate an issue
 
-## Use case 1: Know the problem service, not the cause
+Profiling data is idea when you know there is a specific service or area where there is a performance issue.
+Maybe an alert from spike in CPU led you to profiles or maybe your logs showed OOM issue for a particular service and you need to debug it.
+Explore Profiles lets you quickly drill into a service and identify the performance issue in these scenarios.
 
-If you know the affected service (use case 1), your investigation starts by viewing that service using the **Services** view and reviewing the profiles for that service.
+### Example: Know the problem service, not the cause
+
+If you know the affected service (use case 1), your investigation starts by viewing that service using the **All services** view and reviewing the profiles for that service.
 In the first scenario, you may have identified that the `checkoutservice` has an issue.
-You can select `checkoutservice` and then use the **Profile types** view to examine all profiles for that service.
+You can use **Search services** to locate `checkoutservice` and then use the **Profile types** view to examine all profiles for that service.
 
-## Use case 2: Know there is an issue, need to investigate
+### Example: Know there is an issue, need to investigate
 
 If you only know there is an issue and have to investigate, then your investigation starts by using the **All services** exploration type.
```

Comment:
We should stick to "view" and not "exploration type" as the user will understand better IMO.